### PR TITLE
WORKSPACE patch

### PR DESCRIPTION
**Pull request by sadaf-matinkhoo**
Add rbe config to WORKSPACE

This is required to make the sample tests pass.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -13,6 +13,18 @@
 # limitations under the License.
 workspace(name = "bazel_toolchains")
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rbe_default",
+    # Change the sha256 digest to the value of the `configs_tarball_digest` in the manifest you
+    # got when you ran the curl command above.
```
Comment (smukherj1) [inline WORKSPACE:21]: Remove

File: WORKSPACE [hunk 2]
```diff
@@ -13,6 +13,18 @@
...
+    sha256 = "9f81180099ebc84da906c53cf56c1e9835d7dbc4bbc8ac5c7e075f050c450c3a",
+    # Change "bazel_4.0.0" in the URL below with whatever "bazel_<version>" you downloaded the
+    # manifest for in the previous step.
```
Comment (smukherj1) [inline WORKSPACE:24]: Remove

File: WORKSPACE [hunk 3]
```diff
@@ -13,6 +13,18 @@
 # limitations under the License.
 workspace(name = "bazel_toolchains")
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```
Comment (smukherj1) [inline WORKSPACE:18]: Mention this uses pre-generated configs for the latest available Bazel version and Ubuntu 16.04 toolchain container. Point to the section of the README you copied this from.

Also mention the link is actually unstable (i.e., sha256 digest may change without notice). Also, pre-generated configs are provided on a best-effort basis and don't have any availability guarantees and shouldn't be used in production systems. Point to the README on how users can generate their own configs.

File: WORKSPACE [hunk 4]
```diff
@@ -13,6 +13,18 @@
...
+    name = "rbe_default",
+    # Change the sha256 digest to the value of the `configs_tarball_digest` in the manifest you
+    # got when you ran the curl command above.
+    sha256 = "9f81180099ebc84da906c53cf56c1e9835d7dbc4bbc8ac5c7e075f050c450c3a",
```
Comment (smukherj1) [inline WORKSPACE:22]: Remove. See comment above for why. Maybe add a comment here explaining why we didn't include a sha256 digest.

File: WORKSPACE (hunk 3 above)
Comment (sadaf-matinkhoo) [reply WORKSPACE:27]: Done.

File: WORKSPACE [hunk 5]
```diff
@@ -13,6 +13,25 @@
...
 
+# Use pre-generated toolchain configs the latest Bazel and latest Ubuntu 16.04
+# container. Pre-generated configs are only provided as a convenience for
+# experimenting with configuring Bazel for remote builds. Further, there are
+# no guarantees on how long after a new release of Bazel or the Ubuntu 16.04
+# container mentioned above the corresponding pre-generated configs will be
+# available. So, never depend directly on the URL mentioned below to download
+# toolchain configs in production because they may break without warning.
+# For more information and alternatives, please visit:
+# https://github.com/bazelbuild/bazel-toolchains
```
Comment (smukherj1) [inline WORKSPACE:24]: nit: Change to https://github.com/bazelbuild/bazel-toolchains#rbe_configs_gen---cli-tool-to-generate-configs to be a bit more specific.

File: WORKSPACE [hunk 6]
```diff
@@ -13,6 +13,25 @@
 # limitations under the License.
 workspace(name = "bazel_toolchains")
 
+# Use pre-generated toolchain configs the latest Bazel and latest Ubuntu 16.04
```
Comment (smukherj1) [inline WORKSPACE:16]: nit: configs for

**Review by smukherj1 — APPROVED** [review]

**Pull request merged**